src/routes/api/v1/metadata-map/+server.ts

```diff
@@ -5,6 +5,7 @@ import { loadPageMetadataMap } from "$lib/loadPageMetadataMap";
 export const GET = (async ({ locals: { contentfulClient } }) => {
   const { pageMetadataMap } = await loadPageMetadataMap({
     includeBreadcrumbs: false,
+    includeRedirects: false,
```

Comment:
Unsure if this should go here; I think this API endpoint was added after I left? I'm unsure how / where it's being used.  Does this help power search, preview, or something else?

Reply:
If I recall correctly, this endpoint was just added as a utility endpoint of sorts as a way to quickly view the metadata map via a request. I think I used it to assist building out search. It's not an active dependency for any other feature. I think it's a nice thing to keep around as a quick way to check those details when needed.

Reply:
Ah okay, that makes sense. Thanks for the explanation! I don't think it's harming anything, so I'm cool with keeping it around.